homework_postgres/PostgreSQL-Students.ipynb

```diff
@@ -40,9 +40,22 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.                                    INNER JOIN roles ON person.id = roles.person_id
</pre><p>Inner Join here is correct answer, but why? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/haicheviet/dataeng-course-1/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='140'/>